src/overrides.jl

```diff
@@ -144,3 +144,28 @@ function induced_subgraph(g::T, vlist::AbstractVector{U}) where T <: AbstractSim
     newg.weights = new_weights
     return newg, Vector{E}(vlist)
 end
+
+function induced_subgraph(g::T, elist::AbstractVector{U}) where T <: AbstractSimpleWeightedGraph where U <: AbstractEdge
+    allunique(elist) || throw(ArgumentError("Edges in subgraph list must be unique"))
+    E = eltype(g)
+    W = edgetype(g)
+    vertexSet = Set{E}()
```

Comment:
For style coherence reasons in Julia we would prefer `vertex_set` (same for `vertex_list`)

Reply:
Done!